### DISCUSSION TITLE
请求后端服务时404

### DISCUSSION THREAD

**Discussion by dylan8888** [apache/apisix#8455] [Q&A]
遇到一个很奇怪的现象，暂时无法确定哪个环境出问题。就是后端服务和apisix都是部署在k8s上，前端调用apisix网关接口时，偶尔返回404。不知道大家有没有遇到过，然后大概是哪个环节的问题？

**Comment by tzssangglass** [comment]
1. config `show_upstream_status_in_response_header ` as true, ref: https://github.com/apache/apisix/blob/master/docs/zh/latest/debug-function.md
2. What version of APISIX is being used?
3. Whether to handle a mix of http and https traffic?

**Reply by dylan8888** [reply]
we use docker image is apache/apisix:2.14.1-alpine。and just http protocol

**Reply by tzssangglass** [reply]
You can find the access log corresponding to the 404 response and see if it came from upstream.

**Comment by dylan8888** [comment]
您好，您的邮件我已经收到，我会尽快阅读和回复，谢谢您的邮件。
